UnityAds/src/main/java/com/mopub/mobileads/UnityInterstitial.java

```diff
@@ -128,28 +130,16 @@ protected void show() {
             return;
         }
 
-        if (UnityAds.isReady(mPlacementId)) {
-            // Lets Unity Ads know when ads succeeds to show
-            MediationMetaData metadata = new MediationMetaData(mContext);
-            metadata.setOrdinal(++impressionOrdinal);
-            metadata.commit();
-
+        if (mLoadedPlacementId == null) {
             UnityAds.show((Activity) mContext, mPlacementId, mUnityShowListener);
-        } else {
-            // Lets Unity Ads know when ads fail to show
-            MediationMetaData metadata = new MediationMetaData(mContext);
-            metadata.setMissedImpressionOrdinal(++missedImpressionOrdinal);
-            metadata.commit();
+            return;
```

Comment:
Actually. at this stage, `mLoadedPlacementId` and `mPlacementId` should be the same (unless not loaded before).
So we could simply remove the `if` statement. However, what was the previous call to `metadate.setMissedImpressionOrdinal` doing? Do we still need it?